Backpack-SwiftUI/Calendar/Classes/Core/CalendarMonthGrid.swift

```diff
@@ -30,8 +30,8 @@ struct CalendarMonthGrid<
 
     @State private var dayCellHeight: CGFloat = 0
     @ViewBuilder let dayCell: (Date) -> DayCell
-    @ViewBuilder let emptyLeadingDayCell: (EmptyCellInfo) -> EmptyLeadingDayCell
```

Comment:
Empty cell info handled internally, as it's not something of interes outside this particualr view.